packages/wallets/src/features/cashier/helpers/helpers.ts

```diff
@@ -0,0 +1,86 @@
+import { THooks, TMarketTypes, TWalletLandingCompanyName } from '../../../types';
+import { appIconMapper, walletIconMapper } from '../constants';
+
+type TGetAccountNameProps = {
+    accountCategory: THooks.TransferAccount['account_category'];
+    accountType: THooks.TransferAccount['account_type'];
+    displayCurrencyCode?: THooks.CurrencyConfig['display_code'];
+    landingCompanyName: TWalletLandingCompanyName;
+    mt5MarketType?: TMarketTypes.SortedMT5Accounts;
+};
+
+export const getMarketType = (mt5Group?: string) => {
+    if (mt5Group?.includes('financial')) return 'financial';
+    if (mt5Group?.includes('synthetic')) return 'synthetic';
+    if (mt5Group?.includes('all')) return 'all';
+    return 'all';
+};
+
+export const getLandingCompanyNameOfMT5Account = (mt5Group?: string) => {
+    if (mt5Group?.includes('bvi')) return 'bvi';
+    if (mt5Group?.includes('labuan')) return 'labuan';
+    if (mt5Group?.includes('svg')) return 'svg';
+    if (mt5Group?.includes('vanuatu')) return 'vanuatu';
+    return 'svg';
+};
```

Comment:
Question: Why do we need this. We can get it straight from the api right

Reply:
We need it for `transfer_between_accounts`

Reply:
hey @thisyahlen-deriv unfortunately now BE doesn't return `market_type` and `mt5_landing_compane_name` in `transfer_between_accounts` response. In order to not do extra calculations to get these props from MT5LoginList i asked BE to return `market_type` and `mt5_landing_compane_name` in `transfer_between_accounts` response. There is already card created. After BE will updated the response these 2 functions will be deleted. 
@rostislav-deriv pls leave a comments for these 2 functions accordingly 🙏🏻

Reply:
Those TODO comments are back in place